python/pyspark/pandas/missing/frame.py

```diff
@@ -56,6 +55,9 @@ class MissingPandasLikeDataFrame:
     lookup = _unsupported_function(
         "lookup", deprecated=True, reason="Use DataFrame.melt and DataFrame.loc instead."
     )
+    to_gbq = _unsupported_function(
```

Comment:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_gbq.html

it is deprecated since 2.2.0